src/index.ts

```diff
@@ -12,7 +12,7 @@ type ParseOptions = {
 export async function parseDocs(options: ParseOptions) {
   const packageMode = options.packageMode || 'single';
 
-  const apiDocsPath = options.baseDirectory || path.resolve('./', 'docs', 'api');
+  const apiDocsPath = path.resolve('./', 'docs', 'api');
```

Comment:
```suggestion
  const apiDocsPath = path.resolve(options.baseDirectory, 'docs', 'api');
```

To restore original behavior